Pig Stack Trace
---------------
ERROR 2998: Unhandled internal error. getColNum should not be called on range project

java.lang.AssertionError: getColNum should not be called on range project
	at org.apache.pig.newplan.logical.expression.ProjectExpression.getColNum(ProjectExpression.java:242)
	at org.apache.pig.newplan.logical.expression.ProjectExpression.deepCopy(ProjectExpression.java:547)
	at org.apache.pig.newplan.logical.expression.LogicalExpressionPlan.deepCopy(LogicalExpressionPlan.java:97)
	at org.apache.pig.newplan.logical.rules.MergeForEach$MergeForEachTransformer.transform(MergeForEach.java:222)
	at org.apache.pig.newplan.optimizer.PlanOptimizer.optimize(PlanOptimizer.java:110)
	at org.apache.pig.newplan.logical.relational.LogicalPlan.optimize(LogicalPlan.java:269)
	at org.apache.pig.PigServer.executeCompiledLogicalPlan(PigServer.java:1350)
	at org.apache.pig.PigServer.storeEx(PigServer.java:1011)
	at org.apache.pig.PigServer.store(PigServer.java:974)
	at org.apache.pig.PigServer.openIterator(PigServer.java:887)
	at org.apache.pig.tools.grunt.GruntParser.processDump(GruntParser.java:752)
	at org.apache.pig.tools.pigscript.parser.PigScriptParser.parse(PigScriptParser.java:372)
	at org.apache.pig.tools.grunt.GruntParser.parseStopOnError(GruntParser.java:228)
	at org.apache.pig.tools.grunt.GruntParser.parseStopOnError(GruntParser.java:203)
	at org.apache.pig.tools.grunt.Grunt.exec(Grunt.java:81)
	at org.apache.pig.Main.run(Main.java:608)
	at org.apache.pig.Main.main(Main.java:156)
================================================================================
